packages/test/test-utils/src/localLoader.ts

```diff
@@ -40,21 +40,18 @@ export function createLocalLoader(
 
 /**
  * Creates a detached Container and attaches it.
- * @param documentId - The documentId for the container.
  * @param source - The code details used to create the Container.
  * @param loader - The loader to use to initialize the container.
- * @param urlresolver - The url resolver to get the create new request from.
+ * @param attachRequest - The request to create new from.
  */
 
 export async function createAndAttachContainer(
-    documentId: string,
     source: IFluidCodeDetails,
     loader: ILoader,
-    urlResolver: IUrlResolver,
+    attachRequest: IRequest,
 ): Promise<IContainer> {
     const container = await loader.createDetachedContainer(source);
-    const attachUrl = (urlResolver as LocalResolver).createCreateNewRequest(documentId);
```

Comment:
this was the problem spot where it actually only worked with a LocalResolver eventhough it took an IUrlResolver